organization/_posts/2018-01-24-packaging.md

```diff
@@ -0,0 +1,82 @@
+    No aim of complete convergence between experiments, will note where there are differences in different areas of the community.
+  * Establish working version of [proposed test stack](https://docs.google.com/document/d/1LW8OsTFFA9QwsJ9fASkRoJ2E6Gk3UGnOQIcElCL8UCM/edit), compare against use cases (~March time)
+  * Update [Packaging Technical Note](http://hepsoftwarefoundation.org/notes/HSF-TN-2016-03.pdf) based on work since then (Use Cases, Portage, Test Stack)
+  * Release “(Test) Stack in Box” for CVMFS/Container deployment, could be used as a foundation for small experiments (Spring-Late Summer time)
+* 2018 Events
+  * [WLCG/HSF Workshop 26-29 March (Naples)](https://indico.cern.ch/event/658060/). Noted that early bird registration closes on Feb 16th.
+    Wide agenda, inc. Software Development session, where packaging fits in.
+  * [CHEP 2018, 9-13 July (Sofia)](http://chep2018.org). High level Packaging Abstract submitted. An abstract on Spack/SpackDev from FNAL has also been submitted.
+* News from HSF
+  * [Community White Paper](https://arxiv.org/abs/1712.06982) published (still open for signing)
+  * Note Software Development Chapter and [dedicated paper from Software Development WG on ArXiv](https://arxiv.org/abs/1712.07959)
+* Comments/Questions
+  * Ben has a usecase for Portage (tensorflow on CentOS 6) - will communicate with Guilherme and Graeme about this offline.
+
+## Presentation from Patrick: Spack in HEP Update
+* All CMSSW libs/exes built on SL7 with Spack
+  * Externals added as spack packages + scram xml for runtime paths (run/link)
+  * Many deps missing
+  * Scram sets up runtime
+* FWLite with Spack on MacOS
+  * Relocation (via install_name_tool) took long time
+  * PR to Spack to simplify run of tool
+* MacOS Build Distribution
+  * Distribute without rpath changes
+  * Allow non-admins to install a bundle via a disk image
+  * Use “sparse disk image” using hdiutil: a R/W image that grows/can be compacted as needed
+  * Very simple to setup!
+* Disk Image for FWlite/Fireworks
+  * Homebrew for clang/gfortran/cmake
+  * Spack to build FWlite etc
+  * Script/symlinks to `bin/` path in disk images
+* Also: one with just Homebrew, one with gentoo prefix
+  * Some issues with these to resolve
+* Comments from Guilherme on Portage bootstrap
+  * Uses system compiler to bootstrap a compiler for the prefix - you can use a Mac build of gcc4.9 to get started (will then build gcc more easily)
+  * Bug reports are very welcome
+* Spack binary caches do work, with Partick’s patches
+* Spack have released 0.11 and plan a monthly release cycle from now on
+
+## Presentation from Ben: ABIs and Packaging
+* Binary compatibility could save a lot of build time and space (more re-use)
+  * Raise topic to promote discussion and knowledge sharing
+* C++ ABI compatibility is a complex subject
+  * How package is written
+  * What compiler/standard is used
+* Patching?
```

Comment:
Lines 59-61 were notes to myself for questions I didn't manage to ask before we got kicked out, so I think we should removed them (they're a bit meaningless here).